src/Features/Core/Portable/DocumentSpan.cs

```diff
@@ -57,9 +47,11 @@ public bool CanNavigateTo()
 
         public bool TryNavigateTo()
         {
-            var workspace = Document.Project.Solution.Workspace;
+            var solution = Document.Project.Solution;
+            var workspace = solution.Workspace;
             var service = workspace.Services.GetService<IDocumentNavigationService>();
-            return service.TryNavigateToPosition(workspace, Document.Id, SourceSpan.Start);
+            return service.TryNavigateToPosition(workspace, Document.Id, SourceSpan.Start,
+                options: solution.Options.WithChangedOption(NavigationOptions.PreferProvisionalTab, true));
```

Comment:
Is this right?  We should definitely use the provisional tab on preview if you single click, but do we also use it on double click?